apps/fluent-tester/src/E2E/RadioGroupExperimental/specs/ExperimentalRadioGroup.spec.win.ts

```diff
@@ -1 +1,114 @@
+  });
+
+  it('Radio - Do not set accessibilityLabel -> Default to RadioButton label', async () => {
+    await expect(await RadioGroupExperimentalPageObject.getRBAccessibilityLabel(RadioSelector.Second)).toEqual(SECOND_RADIO_LABEL);
+    await expect(await RadioGroupExperimentalPageObject.didAssertPopup()).toBeFalsy(RadioGroupExperimentalPageObject.ERRORMESSAGE_ASSERT);
+  });
+});
+
+describe('RadioGroup Functional Testing', async () => {
+  /* This resets the RadioGroup state by clicking/selecting the 1st Radio in the RadioGroup */
+  beforeEach(async () => {
+    await RadioGroupExperimentalPageObject.scrollToTestElement();
+    await RadioGroupExperimentalPageObject.waitForPrimaryElementDisplayed(PAGE_TIMEOUT);
+
+    await RadioGroupExperimentalPageObject.resetRadioGroupSelection();
+  });
+
+  it('Click on a Radio and ensure it changes state from unselected -> selected', async () => {
+    /* Validate the Radio is not initially selected */
+    await expect(await RadioGroupExperimentalPageObject.isRadioSelected(RadioSelector.Second)).toBeFalsy();
+
+    /* Click on the Radio to select it */
+    await RadioGroupExperimentalPageObject.clickRadio(RadioSelector.Second);
+    await RadioGroupExperimentalPageObject.waitForRadioSelected(RadioSelector.Second, PAGE_TIMEOUT);
+
+    /* Validate the Radio is selected */
+    await expect(await RadioGroupExperimentalPageObject.isRadioSelected(RadioSelector.Second)).toBeTruthy();
+    await expect(await RadioGroupExperimentalPageObject.didAssertPopup()).toBeFalsy(RadioGroupExperimentalPageObject.ERRORMESSAGE_ASSERT);
+  });
+
+  it('Keyboard to Radio and check for Selection state', async () => {
+    // Presses the ArrowDown key while the first (A) Radio is selected
+    await RadioGroupExperimentalPageObject.sendKey(Keys.ARROW_DOWN, RadioSelector.First);
+    await RadioGroupExperimentalPageObject.waitForRadioSelected(RadioSelector.Second, 5000);
+
+    /* Validate the Radio is selected */
+    await expect(await RadioGroupExperimentalPageObject.isRadioSelected(RadioSelector.Second)).toBeTruthy();
+    await expect(await RadioGroupExperimentalPageObject.didAssertPopup()).toBeFalsy(RadioGroupExperimentalPageObject.ERRORMESSAGE_ASSERT);
+  });
+
+  it("Keyboard to DISABLED Radio and validate it doesn't get selected", async () => {
+    // Presses the ArrowDown key while the second (B) Radio is selected
+    await RadioGroupExperimentalPageObject.sendKey(Keys.ARROW_DOWN, RadioSelector.Second);
+    await RadioGroupExperimentalPageObject.waitForRadioSelected(RadioSelector.Fourth, 5000); // It should skip RadioButton 3 since it is disabled
+
+    /* Validate the Radio is selected */
+    await expect(await RadioGroupExperimentalPageObject.isRadioSelected(RadioSelector.Fourth)).toBeTruthy();
+    await expect(await RadioGroupExperimentalPageObject.didAssertPopup()).toBeFalsy(RadioGroupExperimentalPageObject.ERRORMESSAGE_ASSERT);
+  });
```

Comment:
Let's also add a test case validating circular navigation keyboard behavior

Reply:
And that pressing "tab" navigates outside the RadioGroup

Reply:
Added the test for circular navigation!

Reply:
Added!